pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -344,7 +360,11 @@ func (r *ReconcileJenkinsBaseConfiguration) createServiceAccount(meta metav1.Obj
 	serviceAccount := &corev1.ServiceAccount{}
 	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: meta.Name, Namespace: meta.Namespace}, serviceAccount)
 	if err != nil && apierrors.IsNotFound(err) {
-		serviceAccount = resources.NewServiceAccount(meta, r.Configuration.Jenkins.Spec.ServiceAccount.Annotations)
+		annotations := r.Configuration.Jenkins.Spec.ServiceAccount.Annotations
+		if annotations == nil {
+			annotations = make(map[string]string)
```

Comment:
Shouldn't we add at this step the oauth annotation ?

Reply:
Removing this change currently as route should also be configurable from the CR if it is present.